.rspec

```diff
@@ -1,4 +1,3 @@
 --require rails_helper
 --require spec_helper
---require turnip/rspec
```

Comment:
Can we also uninstall the gem? I think @tonyheadford will be very happy to hear about that

Reply:
Yes. I was looking at producing some features, but it looks like RSpec are implementing their own.